packages/create-hydrogen/src/services/init.ts

```diff
@@ -164,6 +164,24 @@ async function init(options: InitOptions) {
           task.title = 'Completed clean up'
         },
       },
+      {
+        title: 'Initializing a Git repository...',
```

Comment:
@cartogram If you wouldn't mind taking a look at the extra initialization step I added here. In addition, how where you testing the init command locally?

Reply:
Is there also a default `.gitignore` file we should provide?

Reply:
I really like the idea of a default `.gitignore` if we don't provide one already.